### PR TITLE
[NEW] Deploy React on Linode

### PR DESCRIPTION
**Pull request by Jskobos**

### CODE REVIEW

**Comment by EdwardAngert** [comment]
hello, @pbzona 👋 
I'm Edward, and I'll be conducting a copy edit on the guide you've submitted.

I look forward to working with you on this matter.

**Comment by pbzona** [comment]
@EdwardAngert 
Dear Sir:

Hello and thank you for your update on this PR. My happiness to receive your feedback will depend entirely upon the amount of praise included therein. Please be mindful of this while reviewing my guide.

Regards, 
Phil

**Comment by EdwardAngert** [comment]
alas, I regret only that there was so little editing to do.

I appreciate you, and thank you for your continued efforts, Sir @pbzona.


My [PR](https://github.com/Jskobos/docs/pull/1) awaits @Jskobos's approval and merging.

**Comment by pbzona** [comment]
Fantastic, thanks for your service @EdwardAngert 👍

File: docs/development/javascript/_index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+author:
+  name: Linode
+  email: docs@linode.com
+description: ''
```
Comment (GuessWhoSamFoo) [inline docs/development/javascript/_index.md]: This needs a description for SEO purposes

File: docs/development/javascript/deploy-a-react-app-on-linode.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  email: phil.b.zona@gmail.com
+description: 'Learn to deploy a locally developed React application to your Linode using Rsync.'
+og_description: 'Use Rsync to deploy a React application from your local computer to a Linode.'
+keywords: ['react','reactjs','deploy','rsync']
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2018-01-31
+modified: 2018-01-31
+modified_by:
+  name: Linode
+title: "Deploy a React App on Linode"
```
Comment (GuessWhoSamFoo) [inline docs/development/javascript/deploy-a-react-app-on-linode.md]: Application

File: docs/development/javascript/deploy-a-react-app-on-linode.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+2.  Using a text editor, create a deployment script called `deploy` in your app's root directory. Replace `exampleuser` with the username of your limited user account, and `mydomain.com` with your Linode's FQDN or public IP address.
+
+    {{< file "~/myapp/deploy" bash >}}
+#!/bin/sh
+
+echo "Switching to branch master"
+git checkout master
+
+echo "Building app"
+npm run build
```
Comment (GuessWhoSamFoo) [inline docs/development/javascript/deploy-a-react-app-on-linode.md]: Makes the assumption that node is installed locally

**Pull request merged**